docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,101 @@
+*******************
+SupplyChain Example
+*******************
+
+SupplyChain is a example application built on top of Hyperledger Sawtooth.
+SupplyChain demonstrates how various parts of the Sawtooth Architecture can be
+hooked together to provide a full application platform.
+
+In this example provided the infrastructure necessary track physical objects
+thru a real world supply chain on the Sawtooth Ledger. Obviously we can not put
+physical objects in to in to the Digital Ledger. We track Records on the Ledger
+which are uniquely linked to the physical object they represent.
+
+
+SupplyChain Overview
+=====================
+
+Agents
+++++++
+
+Agents represent actors in the system. An Agent is expected most commonly
+to represent a company. Agents are identified by their public key, any
+transactions an agent submits must be signed by the agent. Agents serve two
+roles in the system, owners and custodians. Owners are Agent that are the
+legal holder of the physical object a record represents. Custodians are Agents
+that have possesion of the Physical object that a Record represents. Custodians
+may be transportation, logistics, warehousing companies, or any other company
+that has entered into an agreement to care for the item.
+
+
+The ledger representation of "Agents" is:
+
+.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
+    :language: protobuf
+    :lines: 22-
+
+Records
++++++++
+
+Records, as we discussed above, represent physical objects in the world.
+Records are identified in the system by `natural keys
+<https://en.wikipedia.org/wiki/Natural_key>`_. These natural keys could be the
+serial number of the object, the RFID tag number, etc.
+
+Records keep a list of all the Owners and Custodians that have possesed the
+item, these records are stored in chronological order for oldest to most
+recent. The most recent is the current Agent in that role.
+
+Records may also be finalized. When a record is finalized it means the physical
+item has been consumed or destroyed and there can be no more updates to the
+Record.
+
+The ledger representation of "Records" is:
+
+.. literalinclude:: ../../../../families/supplychain/protos/record.proto
+    :language: protobuf
+    :lines: 22-
+
+
+Applications
+++++++++++++
+
+Applications are the mechanism that Agents change roles(Owner or Custodian).
+Agents that wish to assume a new role create a  new application for the Record.
```

Comment:
Combine these two sentences into one sentence, e.g.:

"Agents that wish to assume a new role for a Record create a new application that specifies the desired role."